less/quicksilver/template.less

```diff
@@ -297,3 +306,8 @@ footer#solstice-footer{
      }
    }
 }
+
+// Hide the default button to manage cookies from cookie-consent
+.cc-revoke {
```

Comment:
We should do this in javascript instead. We should not trigger the pop-up tab if we have a link to manage cookies on the page.